scripts/clean-old-builds.sh

```diff
@@ -38,7 +38,10 @@ if [ "$#" -gt 0 ]; then
     
     MAX_DAYS_SINCE_MODIFIED_SHORT=1
     max_sec_since_modified_short=$(( 3600 * 24 * $MAX_DAYS_SINCE_MODIFIED_SHORT ))
-    
+
+    MAX_DAYS_SINCE_MODIFIED_SHORTEST=0.16
```

Comment:
Because this is a round of of 0.16666667. 
But given this is an arbitrary value, it could be any kind of we think are appropriate. If you prefer it could be 0.1 or 0.2 or 0.5.